ya_httpx_client/service_base.py

```diff
@@ -12,17 +12,17 @@ class ServiceBase(Service):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.queue = self._yhc_cluster.request_queue
+        self.queue = self._yhc_cluster.request_queue  # pylint: disable=no-member
```

Comment:
Why is this needed? Seems like something to be fixed rather than ignored.

Reply:
This is a class created in a dynamic way (because the released `yapapi` is missing instance parametrization), so here we use in the general class an attribute that is defined only in the class that inherits from it.

I don't know how to fix this in a pretty way? Add a class attribute that will be always overwritten? That's not pretty at all.

I would leave this as-is because this will either way change with newer `yapapi`.

Reply:
https://github.com/golemfactory/ya-httpx-client/issues/8